sh/binfmt.sh.in

```diff
@@ -54,7 +54,7 @@ if [ $# -gt 0 ]; then
 else
 	# The hardcoding of these paths is intentional; we are following the
 	# systemd spec.
-	binfmt_dirs='/usr/lib/binfmt.d/ /run/binfmt.d/ /etc/binfmt.d/'
+	binfmt_dirs='/usr/lib/binfmt.d/ /usr/local/lib/binfmt.d/ /run/binfmt.d/ /etc/binfmt.d/'
```

Comment:
We should follow systemd (and general practice, I think) and do `/usr/local` first. We should perhaps tweak the order (in a separate, first commit) to follow the order for existing directories too, per https://www.freedesktop.org/software/systemd/man/latest/binfmt.d.html.

Reply:
I think the reverse order is here on purpose: later ones override earlier ones. This is a bit unlike systemd, where earlier ones are applied first, and later ones are applied only if not specified before.

Reply:
Ah, I see! Thanks.